site/hi.njk

```diff
@@ -0,0 +1,123 @@
+---
+layout: default
+meta_title: COVID-19 महामारी को रोकने के लिए एक आंदोलन
+meta_description: हमारी सरकारें इसे अकेले नहीं कर सकती हैं, हमें अभी कार्रवाई करनी है! अपना हिस्सा करो और भाड़ में जाओ घर!
+body_class: home
+title: '#StayTheFuckHome'
+lang: hi
+---
+
+<main id="main" class="main inner flex flex-1 flex-col py-10 lg:py-20 focus:outline-none" tabindex="-1">
+
+    <section id="container-centre" class="column centre flex-1">
+
+        <h1 class="page-title">{{ title }}</h1>
+        <h2 class="page-subtitle">COVID-19 महामारी को रोकने के लिए एक आंदोलन</h2>
+
+        <div class="content">
+
+            <p class="text-xl">
+
+            हमारी सरकारें केवल धीरे-धीरे <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2"> SARS-CoV-2 </a> के प्रसार और <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019"> COVID-19 महामारी </a> को नियंत्रित करने के उपायों को लागू कर रही हैं । धीमी प्रतिक्रियाओं, सार्वजनिक तुष्टिकरण की नीतियों और अर्थव्यवस्था को स्थिर करने के उनके आग्रह ने उन्हें इस बीमारी से बचाने के लिए आवश्यक उपायों को करने से रोक दिया है। हालाँकि, यह न केवल सरकार का बोझ है। इस धरती के नागरिकों के रूप में, अब कार्रवाई करने और COVID -19 से लड़ने में अपनी ओर से काम करने का समय है।
```

Comment:
`हमारी सरकारें केवल धीरे-धीरे <a href="https://en.wikipedia.org/wiki/Severe_acute_respiratory_syndrome_coronavirus_2"> SARS-CoV-2 </a> के प्रसार और <a href="https://en.wikipedia.org/wiki/Coronavirus_disease_2019"> COVID-19 महामारी </a> को नियंत्रित करने के उपायों को लागू कर रही हैं । धीमी प्रतिक्रियाओं, सार्वजनिक तुष्टिकरण की नीतियों और अर्थव्यवस्था को स्थिर करने के उनके आग्रह ने उन्हें इस बीमारी से बचाने के लिए आवश्यक उपायों को करने से रोक दिया है। हालाँकि, यह केवल सरकार का बोझ नहीं है। इस धरती के नागरिकों के रूप में, अब कार्रवाई करने और COVID -19 से लड़ने में अपनी ओर से काम करने का समय है।
`

Edited a grammatical error in the above paragraph.

Reply:
Done. Thank you.